third_party/pause/Makefile

```diff
@@ -1,13 +0,0 @@
-pause: pause.asm
```

Comment:
No, we have a different form under build/ now
On Jun 28, 2016 8:54 PM, "Daniel Smith" <notifications@github.com> wrote:

> In third_party/pause/Makefile
> <https://github.com/kubernetes/kubernetes/pull/28201#discussion_r68881713>
> :
>
> > @@ -1,13 +0,0 @@
> > -pause: pause.asm
>
> did this get moved to contrib?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/28201/files/b3d6bf0dbec0ecfa6c334c90aff8affecdaf508c#r68881713>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AFVgVOIu_aybcmhDoocZNtnII0-uCPGfks5qQeyEgaJpZM4JAvz0>
> .
>